synapse_data_warehouse/synapse_raw/tasks/V2.14.0__shift_all_tasks_under_refresh.sql

```diff
@@ -0,0 +1,17 @@
+USE ROLE accountadmin;
+use schema {{database_name}}.synapse_raw; --noqa: JJ01,PRS,TMP,CP01
+ALTER TASK refresh_synapse_warehouse_s3_stage_task SUSPEND;
+ALTER TASK aclsnapshots_task UNSET schedule;
+ALTER TASK aclsnapshots_task ADD AFTER refresh_synapse_warehouse_s3_stage_task;
+ALTER TASK TEAMSNAPSHOTS_TASK UNSET schedule;
+ALTER TASK TEAMSNAPSHOTS_TASK ADD AFTER refresh_synapse_warehouse_s3_stage_task;
+ALTER TASK VERIFICATIONSUBMISSIONSNAPSHOTS_TASK UNSET schedule;
+ALTER TASK VERIFICATIONSUBMISSIONSNAPSHOTS_TASK ADD AFTER refresh_synapse_warehouse_s3_stage_task;
+ALTER TASK USERGROUPSNAPSHOTS_TASK UNSET schedule;
+ALTER TASK USERGROUPSNAPSHOTS_TASK ADD AFTER refresh_synapse_warehouse_s3_stage_task;
+
+ALTER TASK aclsnapshots_task RESUME;
+ALTER TASK TEAMSNAPSHOTS_TASK RESUME;
+ALTER TASK VERIFICATIONSUBMISSIONSNAPSHOTS_TASK RESUME;
+ALTER TASK USERGROUPSNAPSHOTS_TASK RESUME;
+ALTER TASK refresh_synapse_warehouse_s3_stage_task RESUME;
```

Comment:
Is there any possible issues setting the s3 task to RESUME last? (I'm not too knowledgeable om this part of snowflake)

Reply:
Good question. Snowflake tasks are finicky.  They have to start and resume in the opposite order. Take this mermaid diagram

```mermaid
flowchart  TD
    A --> B
```

To modify task B, I have to stop task A first, then task B.  Then I have to resume task B then task A.  At least last I checked, it was like this - maybe they updated snowflake tasks